server/serverMethods.js

```diff
@@ -15,8 +15,12 @@ var SwaggerBP = require('swagger-boilerplate');
 var C = require('./serverConstants');
 var configLoader = require('./configLoader');
 var FirebaseArchives = require('./firebaseArchives');
+var plivo = require('plivo');
+var GoogleAuth = require('google-auth-library');
 
 
+var dialedNumbersInfo = {}; // Maps dialed SIP numbers to OpenTok session IDs and connection IDs
```

Comment:
Is this safe (from destruction by heroku dyno)? Does this need to be stored in redis?

Reply:
@jeffswartz I've spoken with Antonio and this won't last 5 minutes in Heroku. We'll have to use the persistence. I'll see if I can do that now.

Reply:
See what you think of this: https://github.com/jeffswartz/OpenTokRTC-V2/pull/6/files
I'm slightly worried about the changes of the hangup callback not being called and the key persisting in redis, blocking the number from being called again.